org.gnome.Papers.json

```
@@ -39,6 +39,25 @@
         "*.a"
     ],
     "modules": [
+        {
+            "name": "gdk-pixbuf",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuiltin_loaders=jpeg,png,bmp",
+                "-Dgtk_doc=true",
+                "-Dman=false"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gdk-pixbuf/2.42/gdk-pixbuf-2.42.9.tar.xz",
+                    "sha256": "28f7958e7bf29a32d4e963556d241d0a41a6786582ff6a5ad11665e0347fc962"
```

So that there's no need to manually check this for updates, we can rely on the Flatpak External Data Checker with something like

```suggestion
                    "sha256": "28f7958e7bf29a32d4e963556d241d0a41a6786582ff6a5ad11665e0347fc962",
                    "x-checker-data": {
                        "type": "gnome",
                        "name": "gdk-pixbuf",
                        "stable-only": true
                    }
```